presto-docs/src/main/sphinx/release/release-0.246.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.246
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug introduced in :pr:`15313` that would cause queries to fail when using upper case in SQL function catalog schema names.
+* Fix a possible int overflow error when spilling to temp storage.
+* Fix an issue where prepared statements would allow some non-constant parameters.
+* Fix injection issue when using function namespace manager.
+* Add 30 second minimum to `query.min-expire-age` configuration parameter.
+* Add listener-based revocation model for spilling strategy PER_TASK_MEMORY_THRESHOLD.
+* Disable spill to disk for join queries where the probe side uses grouped execution and the build does not.  Previously these queries would fail with generic_internal_errors.
+* Disallow ORDER BY literals used with Window functions as it's not useful, expensive and most often used wrongly.
+* Support fragment result caching for queries with local exchange (e.g. intermediate aggregation).
+* Optimize queries that have unnecessary limits and order bys.
+  This feature is enabled by default and can be disabled by using the configuration property optimizer.skip-redundant-sort or session property skip_redundant_sort
```

Comment:
Add a period (".") at the end of the line.

Reply:
back quote properties